vendor/src/golang.org/x/net/proxy/direct.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
awesome just wanted to be sure :) you rock

On Fri, Dec 4, 2015 at 1:23 PM, Nathan LeClaire <notifications@github.com>
wrote:

> In vendor/src/golang.org/x/net/proxy/direct.go
> <https://github.com/docker/docker/pull/18373#discussion_r46733723>:
>
> > @@ -0,0 +1,18 @@
> > +// Copyright 2011 The Go Authors. All rights reserved.
>
> @jfrazelle <https://github.com/jfrazelle> Yes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18373/files#r46733723>.
>